spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/repository/query/SpannerQueryLookupStrategyTests.java

```diff
@@ -174,9 +174,13 @@ void getChildrenRowsQueryTest() {
             new SpannerWriteConverter(),
             this.spannerMappingContext);
     assertThat(statement.getSql())
-        .isEqualTo(
-            "SELECT deleted, id3, id, id_2 FROM child_test_table WHERE ((id = @tag0 AND id_2 ="
-                + " @tag1)) AND (deleted = false)");
+        .contains("SELECT ")
+        .contains("deleted")
+        .contains("id3")
+        .contains("id_2")
+        .contains("id")
```

Comment:
This _may_ cause us to under-assert on the sql string returned as it appears to be verifying whether or not "deleted" or "id3" are present but doesn't enforce a query structure.